invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/serializer.js

```diff
@@ -2,12 +2,18 @@ import _isEmpty from 'lodash/isEmpty';
 import { fromISO } from '../date';
 
 function serializeResponse(hit) {
-  const result = { ...hit.metadata };
+  let result = {};
```

Comment:
just for info, not to change, but you can keep `const` because you are not re-assigning the variable. I am in general a fan of consts :) (towards immutability)

Reply:
I agree but here we mutate the object after so having let, even if in objects assignment you can still use const, is more appropriate I think. But in general yes I agree with the `const` usage :)